src/Microsoft.Health.Dicom.Core/Models/Operations/DicomOperation.cs

```diff
@@ -23,5 +23,10 @@ public enum DicomOperation
     /// <summary>
     /// Specifies an export operation that copies data out of the DICOM server and into an external data store.
     /// </summary>
-    Export
+    Export,
+
+    /// <summary>
+    /// Specifies an update operation that updates the Dicom attributes.
+    /// </summary>
+    Update
```

Comment:
I don't see a swagger file generated?

Reply:
Please check now. I have updated the new swagger